lib/util.js

```diff
@@ -194,6 +194,7 @@ function inspect(obj, opts) {
   if (ctx.colors) ctx.stylize = stylizeWithColor;
   if (ctx.maxArrayLength === undefined) ctx.maxArrayLength = kDefaultMaxLength;
   if (ctx.maxArrayLength === null) ctx.maxArrayLength = Infinity;
+  if (ctx.breakLength === undefined) ctx.breakLength = 60;
```

Comment:
what about setting to `null` or `Infinity`... what affect would that need to have? Just thinking about consistency with other options